awx/main/scheduler/task_manager.py

```diff
@@ -277,16 +275,6 @@ def start_task(self, task, rampart_group, dependent_tasks=None, instance=None):
                 task.send_notification_templates('running')
                 logger.debug('Transitioning %s to running status.', task.log_format)
                 schedule_task_manager()
-            elif not task.supports_isolation() and rampart_group.controller_id:
```

Comment:
@jbradberry just FYI, as you find yourself working on isolated node code cleanup/removal, you may find conflicts in this general area.